unitTests/ui/MissingTermsTest.ts

```diff
@@ -123,6 +123,42 @@ export function MissingTermsTest() {
       };
 
       describe('English', () => {
+        describe('when a words contains words boundary character', () => {
+          it(`when the missing term is part of the words,
+          it will not be displayed as a missing term.`, () => {
+            const wordBoundarysCharacter = [
+              `-`,
+              `'`,
+              `?`,
+              `*`,
+              `’`,
+              `.`,
+              `~`,
+              `=`,
+              `,`,
+              `/`,
+              `\\`,
+              `:`,
+              `\``,
+              `;`,
+              `_`,
+              `!`,
+              `&`,
+              `(`,
+              `)`
+            ];
+            let query: string;
```

Comment:
I don't think this variable needs to be out of the `forEach` scope. Would a `const` on line 153 work?